### PR TITLE
APP-250: Improve test suite stability

### PR DESCRIPTION
**Pull request by hedvigoscar**
- Test stability fixes
- Make tests build again
- Update dependency
- Format a file
- Repair some malfunctioning tests
- Fix incorrect resources being loaded on `hasText`-assert
- Prevent app from crashing when `GeoQuery` fails
- Repair tests using WebSocket connections
- Prevent Mixpanel from crashing our tests
- Fix the last test instability issues

### Checklist

- [X] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/androidTest/java/com/hedvig/app/util/KakaoExtensions.kt [hunk 1]
```diff
@@ -21,7 +24,16 @@ fun KTextInputLayout.hasError(@StringRes resId: Int, vararg formatArgs: Any) =
 fun KDatePicker.setDate(date: LocalDate) = setDate(date.year, date.monthValue, date.dayOfMonth)
 
 fun KTextView.hasText(@StringRes resId: Int, vararg formatArgs: Any) =
-    hasText(context().getString(resId, *formatArgs))
+    view.check { view, noViewFoundException ->
+        if (view is TextView) {
+            val text = view.resources.getString(resId, *formatArgs)
+            if (text != view.text) {
```
Comment (hugokallstrom) [inline app/src/androidTest/java/com/hedvig/app/util/KakaoExtensions.kt:30]: awesome

**Review by hugokallstrom — APPROVED** [review]
Brilliant. Sorry for not fixing the storyTitle parameter :(

**Comment by hedvigoscar** [comment]
Worry not, these things happen when the test suite has been red for too long. It's the boy who cried wolf-effect 🙁

File: app/src/androidTest/java/com/hedvig/app/util/KakaoExtensions.kt (hunk 1 above)
Comment (hedvigoscar) [reply app/src/androidTest/java/com/hedvig/app/util/KakaoExtensions.kt:30]: I'm not actually sure that it's required any more though given that I defaulted to using a translated context in the utility function 😬

**Pull request merged**